lms/templates/report_modal.html

```diff
@@ -89,6 +89,7 @@
         $("#report-form input[type='submit']").removeAttr("disabled");
         $("#report-error").css("display", "none");
         $("#report-success-wrapper").css("display", "none");
+        $("#report-form-wrapper").css("display", "block");
```

Comment:
Why are you setting the CSS in JS here rather than CSS? Is it completely necessary?